test/unit/spec/ForgotPassword_spec.js

```diff
@@ -350,6 +386,85 @@ function (Q, _, $, OktaAuth, Util, AccountRecoveryForm, PrimaryAuthForm, Beacon,
           });
         });
       });
+      itp('shows an error if username is empty and request Voice Call', function () {
+        return setupWithCall().then(function (test) {
+          $.ajax.calls.reset();
+          test.form.makeCall();
+          expect($.ajax).not.toHaveBeenCalled();
+          expect(test.form.usernameErrorField().length).toBe(1);
+        });
+      });
+      itp('makes a Voice Call', function () {
+        return setupWithCall().then(function (test) {
+          $.ajax.calls.reset();
+          test.setNextResponse(resChallengeCall);
+          test.form.setUsername('foo');
+          test.form.makeCall();
+          return tick();
+        })
+        .then(function () {
+          expect($.ajax.calls.count()).toBe(1);
+          Expect.isJsonPost($.ajax.calls.argsFor(0), {
+            url: 'https://foo.com/api/v1/authn/recovery/password',
+            data: {
+              'username': 'foo',
+              'factorType': 'CALL'
+            }
+          });
+        });
+      });
+      itp('updates appState username after making a Voice Call', function () {
+        return setupWithCall()
+        .then(function (test) {
+          test.setNextResponse(resChallengeCall);
+          test.form.setUsername('foo');
+          test.form.makeCall();
+          return tick(test);
+        })
+        .then(function (test) {
+          expect(test.router.appState.get('username')).toBe('foo');
+        });
+      });
+      itp('shows an error if making a Voice Call results in an error', function () {
+        return setupWithCall()
+        .then(function (test) {
+          Q.stopUnhandledRejectionTracking();
+          test.setNextResponse(resError);
+          test.form.setUsername('foo');
+          test.form.makeCall();
+          return tick(test);
+        })
+        .then(function (test) {
+          expect(test.form.hasErrors()).toBe(true);
+          expect(test.form.errorMessage()).toBe('You do not have permission to perform the requested action');
+        });
+      });
+      itp('does not have a problem with sending email after making a Voice Call', function () {
+        return setupWithCall()
+        .then(function (test) {
+          Q.stopUnhandledRejectionTracking();
```

Comment:
Same here - remove and check if there are errors in the test results or the console. FYI, there should be no errors/logs in the console when you do a full run (as of my last commit I cleaned up all the issues).